Create ui-c8y-1020-9-1-smart-rule-pin-dropdown-to-the-card-body-6553-graft-release.md (#2297)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1020-9-1-smart-rule-pin-dropdown-to-the-card-body-6553-graft-release.md

---
date: ""

> title: Smart rule pin dropdown moved to the card body

Reviewer: ```suggestion
title: Smart rule pin dropdown moved from the header to the card body
```

> title: Smart rule pin dropdown moved to the card body

Reviewer: ```suggestion
title: Smart rule pin dropdown moved from the header to the card body
```

> title: SmartRule_pin_dropdown_to_the_card_body (#6553) [GRAFT][release/cd] (#6759)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Smart rule pin dropdown moved to the card body
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-58122
version: 1020.9.1
---

> SmartRule_pin_dropdown_to_the_card_body (#6553) [GRAFT][release/cd] (#6759)

Author: The Claude AI suggested to use the following body:
```suggestion
To improve the user experience when configuring smart rules, the pin dropdown has been moved from the smart rule header to the card body. This change makes it easier for users to find and interact with the pin dropdown, as it is now located in a more prominent and intuitive position within the smart rule configuration interface. The new placement of the pin dropdown is expected to enhance usability and reduce confusion for users when setting up and managing their smart rules.
```

> To improve the user experience when configuring smart rules, the pin dropdown has been moved from the smart rule header to the card body. This change makes it easier for users to find and interact with the pin dropdown, as it is now located in a more prominent and intuitive position within the smart rule configuration interface. The new placement of the pin dropdown is expected to enhance usability and reduce confusion for users when setting up and managing their smart rules.

Reviewer: ```suggestion
To improve the user experience when configuring smart rules, the pin dropdown has been moved from the smart rule header to the card body. This change makes it easier for users to find and interact with the pin dropdown, as it is now located in a more prominent and intuitive position within the smart rule configuration. 
```
